### ISSUE TITLE
Expose courses via paginated JSON API

### ISSUE THREAD

**Issue by jprberlin** [mammooc/mammooc.org#583]
The current json route has no pagination support, or?

**Comment by MrSerth** [comment]
That's correct: The courses.json is not paginated and I don't know whether all available information is exposed there.
